pkg/kubelet/cm/helpers_linux.go

```diff
@@ -44,3 +154,36 @@ func GetCgroupSubsystems() (*CgroupSubsystems, error) {
 		MountPoints: mountPoints,
 	}, nil
 }
+
+// getCgroupProcs takes a cgroup directory name as an argument
+// reads through the cgroup's procs file and returns a list of tgid's.
+// It returns an empty list if a procs file doesn't exists
+func getCgroupProcs(dir string) ([]int, error) {
+	procsFile := filepath.Join(dir, "cgroup.procs")
+	_, err := os.Stat(procsFile)
```

Comment:
you can get rid of this and do the `IsNotExist` check on the result from `os.Open`.

It's less racy too for the case that the file vanishes after you stat it.